tool-populate/api/Roles.js

```diff
@@ -23,13 +24,12 @@ function loadRoles(options, callback) {
 }
 
 function createRole(roleToCreate) {
-  var roleACL = new Parse.ACL();
-  roleACL.setPublicReadAccess(true);
-
+  var acl = new Parse.ACL()
+  acl.setPublicReadAccess(false)
```

Comment:
If the role objects are explicitly restricted `read` then queries to Parse.Role within the client will likely return an empty set. See https://github.com/ecohealthalliance/GoodQuestion/blob/master/js/api/Submissions.js#L45-L54